Shared/Include/LinearCurve.hpp

```diff
@@ -0,0 +1,15 @@
+//
+// Created by hippolyteb on 11/24/16.
+//
+
+#ifndef R_TYPE_LINEARCURVE_HPP
+#define R_TYPE_LINEARCURVE_HPP
+
+#include "ITweeningCurve.hpp"
+
+class LinearCurve : public ITweeningCurve {
+    double Curve(double const &d) const override final;
```

Comment:
My bad ! `final`modifier isn't set automatically by CLion. In term of bytecode output it makes no difference.